Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,59 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
 
 namespace Logger.Tests
 {
     [TestClass]
     public class FileLoggerTests
     {
-        
+        [TestMethod]
+        public void Log_CreateFile()
+        {
+            // Arrange
+            string filePath = Path.GetRandomFileName();
+
+            try
+            {
+                // Act
+                FileLogger fileLogger = new FileLogger(filePath);
+                fileLogger.Debug("Debug Message");
+
+                // Assert
+                Assert.IsTrue(File.Exists(filePath));
+            }
+            finally
+            {
+                File.Delete(filePath);
+            }
+        }
+
+        [TestMethod]
+        public void Log_LogMessage()
+        {
+            // Arrange
+            string filePath = Path.GetRandomFileName();
+
+            try
+            {
+                // Act
+                FileLogger fileLogger = new FileLogger(filePath);
+                fileLogger.Debug("Debug Message {0}", 4);
+                fileLogger.Information("Information Message {0}", 3);
+                fileLogger.Warning("Warning Message {0}", 2);
+                fileLogger.Error("Error Message {0}", 1);
```

Comment:
This is fine, but since the extension methods are not what are being tested here, a single call to Log would suffice. It also obscures the fact that LogMessage is what is being tested (according to the test method name), but LogMessage does not appear inside of the unit test.

Reply:
I must have forgotten I already tested those methods.